Master

jcriadop opened the pull request:
pagination fixed
update from old version fixed
track number fixed
translated vars to english

jcriadop committed:
Revert "Update config.xml"

This reverts commit 914df27a670667eee548702dfc748605f4351a6f.

jcriadop committed:
bugs fixed

pagination fixed
update from old version fixed
track number fixed
translated vars to english

jcriadop committed:
Revert "bugs fixed"

This reverts commit 820dc9b8458b908add7d15f6098db2fd07f38715.

jcriadop committed:
Revert "Revert "bugs fixed""

This reverts commit 3119421587883bf6fd01b3489a888a83d800a51d.

jcriadop committed:
Revert "Revert "Revert "bugs fixed"""

This reverts commit c6be4fd3762429620ab12137570422f7ee2d6a09.

jcriadop committed:
Revert "Revert "Revert "Revert "bugs fixed""""

This reverts commit f07a61d9fe7504f8e6ba17e4925c84104d5e70a4.

Quetzacoalt91 commented on upgrade/upgrade-1.3.php:
```diff
@@ -0,0 +1,170 @@
+	
+	if (in_array('ZELERIS_DESCRIPCION_MERCANCIA',$config_installed))
+	{
+		$sql='UPDATE '._DB_PREFIX_.'configuration SET `name` = "ZELERIS_MERCHANDISE_DESCRIPTION" WHERE `name`="ZELERIS_DESCRIPCION_MERCANCIA"';
```
You should update your data like this (assuming you want to remove the old fields:

```php
Configuration::updateValue('ZELERIS_MERCHANDISE_DESCRIPTION', Configuration::get('ZELERIS_DESCRIPCION_MERCANCIA'));
Configuration::deleteByName('ZELERIS_DESCRIPCION_MERCANCIA');
```

Quetzacoalt91 commented on zeleriscarrier.php:
```diff
@@ -41,7 +41,7 @@ public function __construct()
 			$this->name = 'zeleriscarrier';
 			$this->tab = 'shipping_logistics';
-			$this->version = '1.2.5';
+			$this->version = '1.3';
```
The new version should be 1.3.0

Quetzacoalt91 commented on config.xml:
```diff
@@ -1,12 +1,11 @@
+        <module>
+            <name>zeleriscarrier</name>
+            <displayName><![CDATA[Zeleris transporte]]></displayName>
+            <version><![CDATA[1.3]]></version>
```
The new version should be 1.3.0

Quetzacoalt91 commented on upgrade/upgrade-1.3.php:
```diff
@@ -0,0 +1,170 @@
+if (!defined('_PS_VERSION_'))
+	exit;
+
+function upgrade_module_1_3($module)
```
The function name should be ```upgrade_module_1_3_0```

jcriadop committed:
Minor fix

Changed version from 1.3 to 1.3.0
Changed update configuration vars way like required

Quetzacoalt91 commented on upgrade/upgrade-1.3.php:
```diff
@@ -0,0 +1,170 @@
+		}
+	}
+	
+	return $module;
```
You must return a boolean. `true` if the upgrade succeed, or `false` if something went wrong.

Quetzacoalt91 commented:
Please also rename your file *upgrade-1.3.php* => *upgrade-1.3.0.php*

jcriadop committed:
Minor fix

Deleted Big non-used image
Modify upgrade file name
Modify return to boolean on upgrade file

jcriadop commented:
File renamed

Quetzacoalt91 commented:
Hi @jcriadop,

I do not understand what you did in you last commit. Do you know that the upgrade file will be executed one time ? So, if you return directly after the first SQL request execution, you will never be abble to apply the other checks.

Best regards

jcriadop committed:
minor fix

return false fixed

jcriadop commented:
Hi, sorry for mistake, please review now.

Best regards.

Quetzacoalt91 commented:
Hi @jcriadop,

Thanks, your PR has been _manually_ merged.
Even if it is not shown as merged, your commits appear in the master branch: https://github.com/PrestaShop/zeleriscarrier/compare/914df27a670667eee548702dfc748605f4351a6f...f0b991aa9d0ee24aaca718f75c86d7627453ab42

I rebased them in order to remove `Revert "Revert "Revert "Revert "..."""""`.

Best regards

jcriadop commented:
Ok, thanks.
